tests/upgrade/postgres_run.sh

```diff
@@ -83,8 +83,10 @@ test_upgrade_paths() {
 
     EARLIER_SHA="fe924fce30bbec4dbd37d731ccd505837a2c2575"
     EARLIER_TAG="3.74.0-1-gfe924fce30"
-    # To test we remain backwards compatible rollback to 4.1.1
-    FORCE_ROLLBACK_VERSION="4.1.1"
+    # To test we remain backwards compatible rollback to 4.1.x
+    # TODO(ROX-18861): Replace the rollback version to next 4.1 release
+    # We use an engineering build for now until it is available
+    FORCE_ROLLBACK_VERSION="4.1.2-2-g8489ac9584"
```

Comment:
I'm concerned this will break because this image might have an expiration

Reply:
Actually just checked and 
```
 "quay.expires-after": "never",
 ```
 so this works

Reply:
yeah i had been burnt by that before, I was going to check that once this worked.  Now I don't have to